source/association_basics.md

```diff
@@ -950,13 +950,10 @@ NOTE: Опцию `:counter_cache` необходимо указывать тол
 
 #### `:dependent`
 
-Устанавливает, что произойдет со связанными объектами, когда их владелец будет уничтожен:
+Если установить опцию `:dependent`:
```

Comment:
http://edgeguides.rubyonrails.org/association_basics.html#options-for-belongs-to-dependent

?